src/transport/SecurePairingSession.cpp

```diff
@@ -53,14 +54,17 @@ SecurePairingSession::~SecurePairingSession(void)
 }
 
 CHIP_ERROR SecurePairingSession::Init(uint32_t setupCode, uint32_t pbkdf2IterCount, const unsigned char * salt, size_t saltLen,
-                                      SecurePairingSessionDelegate * delegate)
+                                      Optional<NodeId> myNodeId, uint16_t myKeyId, SecurePairingSessionDelegate * delegate)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
 
     VerifyOrExit(salt != nullptr, err = CHIP_ERROR_INVALID_ARGUMENT);
     VerifyOrExit(saltLen > 0, err = CHIP_ERROR_INVALID_ARGUMENT);
     VerifyOrExit(delegate != nullptr, err = CHIP_ERROR_INVALID_ARGUMENT);
 
+    ChipLogProgress(Inet, "Secure Pairing: SetupCode %d, Iter %d, SaltLen %ld", setupCode, pbkdf2IterCount, saltLen);
```

Comment:
I'm wondering if we should be logging this information.  It seems to me like a potential security risk.

Reply:
@jelderton , makes sense. I have remove the logs in the new commit.